### ISSUE TITLE
Version History?

### ISSUE THREAD

**Issue by iRupam** [cdefgah/whatsapp-jpeg-repair#2]
@cdefgah It's a very good piece of software. I hope it gets to more people. I would like you to add a "Version History" in the upcoming releases as most of the software gets a "Version History" tab nowadays.

**Comment by cdefgah** [comment]
@iRupam thank you for feedback, Rupam. 

Every released zip file contains `Changelog.txt` file with information about changes. It is what you are talking about or you are asking for something else?

Thank you.

**Comment by iRupam** [comment]
Yeah, I found it. But it contains only versions from 2.0 to 2.0.1. Is this all? You did not release v1.0 I guess. @cdefgah And yes, I am talking about it.

**Comment by cdefgah** [comment]
@iRupam Version 1.0 was released on February, but I did not keep this release zip file into [releases](https://github.com/cdefgah/whatsapp-jpeg-repair/releases). Because that release was done manually, and since version 2.0 automatic build scripts were introduced and GitHub generates release automatically.

I did not note version 1.0 into `Changelog.txt` as it was the first release of basic and plain version. To prevent any misunderstanding I will mention this version into `Changelog.txt` now.

In case you have feature requests or issue reports feel free to contact me and post issues here.

Thank you for feedback.
